.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Rust
+
+on:
+  push:
+  pull_request:
+
+env:
+  WINFSP_VERSION: "2.0.23075"
+  CARGO_TERM_COLOR: always
+
+jobs:
+  ci:
+    runs-on: windows-latest
+    
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
```

Comment:
Can you use commit SHA instead of the tag ? like we do on parsec-cloud with the _pin comment_